docs/doc/13-monitor/prometheus-and-granafa.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Prometheus & Grafana
+---
+
+[Prometheus](https://prometheus.io/) is a time series database that features a flexible query language and a multi-dimensional data model.
```

Comment:
Databend captures real-time metrics of the meta and query services, which you can access through a web browser using the following URLs:

- Meta Metrics: http://<address>:28101/v1/metrics
- Query Metrics: http://<address>:7070/metrics